From 815f080d52fa2946ad1d4cf4ccd91dd294afa00b Mon Sep 17 00:00:00 2001
From: Andrejs Cainikovs <andrejs.cainikovs@toradex.com>
Date: Mon, 5 Sep 2022 17:51:39 +0200
Subject: [PATCH 209/233] [FIO tonxp] configs: verdin-imx8mm: disable LTO

Yocto calls additional U-Boot target `u-boot-initial-env`,
which fails due to stripped out `.rodata.default_environment`
section from env/common.o when built with LTO enabled.

This commit disables LTO, thus avoiding the build issues.

Upstream-Status: Inappropriate [other]
- NXP downstream specific

Signed-off-by: Andrejs Cainikovs <andrejs.cainikovs@toradex.com>
(cherry picked from commit ec4644888505f69ba82f676129a7f2d237cd470e)
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
---
 configs/verdin-imx8mm_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/verdin-imx8mm_defconfig b/configs/verdin-imx8mm_defconfig
index 36296f726f4..b9d01d8d9f4 100644
--- a/configs/verdin-imx8mm_defconfig
+++ b/configs/verdin-imx8mm_defconfig
@@ -22,7 +22,6 @@ CONFIG_SYS_BOOTCOUNT_SINGLEWORD=y
 CONFIG_SYS_LOAD_ADDR=0x48200000
 CONFIG_SYS_MEMTEST_START=0x40000000
 CONFIG_SYS_MEMTEST_END=0x80000000
-CONFIG_LTO=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_FIT=y
 CONFIG_FIT_EXTERNAL_OFFSET=0x3000
-- 
2.25.1

